### PR TITLE
 testing/py-qrcode: upgrade to 6.0

### PR DESCRIPTION
**Pull request by prspkt**

### CODE REVIEW

**Comment by fabled** [comment]
@fabaff Any comment on this?

File: main/py-pillow/APKBUILD [hunk 1]
```diff
@@ -44,19 +44,7 @@ _py() {
 	install_if="$pkgname=$pkgver-r$pkgrel $python"
 
 	cd "$builddir"
-	$python setup.py install --prefix=/usr --root="$subpkgdir" || return 1
-
-	# The scripts are packages in -doc, they should not be here.
-	rm -r "$subpkgdir"/usr/bin
-}
-
-doc() {
```
Comment (fabaff) [inline main/py-pillow/APKBUILD]: I think that we still should ship the docs. In #3878 I started with this:

```bash
doc() {
	pkgdesc="$pkgdesc (documentation)"
	local docdir="$subpkgdir/usr/share/doc/$pkgname"

	cd "$builddir"/docs
	mkdir -p "$docdir"
	make html
	cp -R _build/html "$docdir"
}
```

Will require Sphinx.

Comment (prspkt) [reply main/py-pillow/APKBUILD]: @fabaff thank you very much for the feedback. Building the html docs for `py-pillow` requires `py-sphinx` and `py-packaging`. The later is in the community repository and travis cannot pull it in as a dependency.

**Comment by andypost** [comment]
Looks good and valid for today

**Comment by ncopa** [comment]
Can you please rebase this and resolve the conflict?

**Comment by prspkt** [comment]
I think we're good.

**Pull request closed without merging**